examples-new/native-authentication-exchange/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "SECRET_KEY": "replace me with a random string",
+  "NYLAS_OAUTH_API_ID": "replace me with the API ID from Nylas",
```

Comment:
In that case, you should probably change it in your developer console, as well. That's where I got the "API ID" and "API Secret" terminology from. I assume you'd also like me to update it in the README files, and everywhere else I've used those terms?
![nylas developer console](https://user-images.githubusercontent.com/132355/28802306-ccb73a6e-7623-11e7-895b-600cd8d0aa6a.png)